src/main/java/no/nav/pto/veilarbportefolje/cv/CVService.java

```diff
@@ -0,0 +1,55 @@
+package no.nav.pto.veilarbportefolje.cv;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import no.nav.arbeid.cv.avro.Melding;
+import no.nav.arbeid.cv.avro.Meldingstype;
+import no.nav.common.types.identer.AktorId;
+import no.nav.pto.veilarbportefolje.elastic.ElasticServiceV2;
+import no.nav.pto.veilarbportefolje.kafka.KafkaConfig;
+import no.nav.pto.veilarbportefolje.kafka.KafkaConsumerService;
+import org.springframework.stereotype.Service;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+
+@RequiredArgsConstructor
+@Service
+@Slf4j
+public class CVService implements KafkaConsumerService<Melding> {
```

Comment:
Kanskje endre litt på nanvet? for nå vil vi ha CVServiceFromAiven som håndterer hjemmel og CVService som håndterer cv innhold. Det føles litt forrvirende 🤔 

Føler selv at det er to veier: en CVservice klasse som håndterer meldinger fra begge topics
eller å rename CVServiceFromAiven til CVHjemmelService

Reply:
Kanskje enklest å forstå med en service på dette :)

Reply:
Will be changed soon to (CVService -> CVHjemmelService)